### PR TITLE
Created the brand new KeyGenErrorScreen

### PR DESCRIPTION
**Pull request by LF0044**
Created the brand new KeyGenErrorScreen.
Fixed PR errors.

Screenshot from emulator
Screenshot_1715099819

### CODE REVIEW

File: app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        ) {
+            Spacer(modifier = Modifier.height(MaterialTheme.dimens.small2))
+
+            TopBar(
+                centerText = stringResource(R.string.keysign),
+                startIcon = null,
+                navController = rememberNavController()
+            )
+
+            Image(
```
Comment (johnnyluo) [inline app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt:53]: Could you please remove this image , we don't need it , even it is in figma , but this is keygen error

Comment (LF0044) [reply app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt:53]: ok

File: app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            .fillMaxSize()
+            .background(MaterialTheme.appColor.oxfordBlue800)
+    ) {
+        Column(
+            modifier = Modifier.align(Alignment.TopCenter)
+        ) {
+            Spacer(modifier = Modifier.height(MaterialTheme.dimens.small2))
+
+            TopBar(
+                centerText = stringResource(R.string.keysign),
```
Comment (johnnyluo) [inline app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt:48]: it is `Keygen` , not `Keysign`

**Comment by johnnyluo** [comment]
image
it should looks like this

File: app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt (hunk 2 above)
Comment (LF0044) [reply app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt:48]: Ok!

But Figma says Keysign:
Screenshot 2024-05-06 at 18 35 09

Comment (LF0044) [reply app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt:48]: Ok!

I think i confused looking at another screen at figma

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**